build.sh

```diff
@@ -16,83 +16,82 @@
 
 # Package build scripts can use this by setting $DESTDIR before launching the
 # script.
-prefix=/opt/google-cloud-ops-agent
+prefix=/usr/libexec/google-cloud-ops-agent
```

Comment:
I don't think we want to do that. The prefix controls the locations of all files, including both the binaries and the non-binary data. It's better to set a separate `bindir` and use that for binaries only.

Reply:
i don't know what non-binary data you mean, but this only affects binaries. here is the contents of the el8 RPM produced by this PR:
```
/etc/google-cloud-ops-agent/config.yaml
/usr/lib/.build-id
/usr/lib/.build-id/82
/usr/lib/.build-id/82/acab8ee480462b5e2460f45fbb892ada1bef3e
/usr/lib/systemd/system-preset/90-google-cloud-ops-agent.preset
/usr/lib/systemd/system/google-cloud-ops-agent-fluent-bit.service
/usr/lib/systemd/system/google-cloud-ops-agent-fluent-bit.service.d
/usr/lib/systemd/system/google-cloud-ops-agent-fluent-bit.service.d/directories.conf
/usr/lib/systemd/system/google-cloud-ops-agent-opentelemetry-collector.service
/usr/lib/systemd/system/google-cloud-ops-agent-opentelemetry-collector.service.d
/usr/lib/systemd/system/google-cloud-ops-agent-opentelemetry-collector.service.d/directories.conf
/usr/lib/systemd/system/google-cloud-ops-agent.service
/usr/libexec/google-cloud-ops-agent/fluent-bit
/usr/libexec/google-cloud-ops-agent/google_cloud_ops_agent_engine
/usr/libexec/google-cloud-ops-agent/otelopscol
```